crates/bevy_ecs/src/event/send_event.rs

```diff
@@ -0,0 +1,15 @@
+use super::{Event, Events};
+use crate::world::{Command, World};
+
+/// A command to Send an arbitrary [`Event`], used by [`Commands::send_event`](crate::system::Commands::send_event).
+pub struct SendEvent<E: Event> {
```

Comment:
Consider adding more bounds to `E` so that `SendEvent` can derive more stuff.
I think `Debug` and `Reflect` are reasonable assumptions. Note that this *may* be controversial.

Reply:
We shouldn't add more bounds here: they're not that useful and pollute things.

Reply:
i disagree with this. the idea is to be ergonomic, adding extra bounds that `Event` doesn't otherwise require seems contrary to that.

Reply:
Fine by me. My suggestion here comes from a place of frequently being annoyed that misc. Bevy types tend to not implement `Debug` and `Reflect`, which is quite annoying when trying to do dynamic editor stuff, or also don't support `PartialEq`, which is annoying in unit tests. But yeah, it's alright for this type :)

Reply:
yeah i definitely agree with the sentiment in general.